### DISCUSSION TITLE
The Skeleton Store is now available! 🎉

### DISCUSSION THREAD

**Discussion by endigo9740** [skeletonlabs/skeleton#1763] [Announcements]
![skeleton-store](https://github.com/skeletonlabs/skeleton/assets/1509726/163535ee-ca00-42f5-86cc-33888149b3ef)

Hey everyone, Chris here from Skeleton Labs. Today I'm thrilled to reveal our newest service, the [Skeleton Store](https://store.skeleton.dev/). A digital storefront featuring premium assets purpose-built by and for Skeleton.

![image](https://github.com/skeletonlabs/skeleton/assets/1509726/d198826b-03b0-4490-9118-7eb5c48efc84)

Today's launch welcomes our first two website templates. This includes the sleek Blog template, as well as modern portfolio website. Each provides a perfect foundation for both beginners and pros alike - whether you lack confidence in your design skills, or just looking for a quick and turnkey solution for deploying a new application.

> 🇺🇸 To our international friends - please note that today's release is initially reserved for customers in the United States. However, we are looking to expand to other countries in the near future. We appreciate your patience in the meantime.

## Adaptive Design

First things first, these are not your tradition website templates – each is designed from the ground up using Skeleton, meaning they're capable of adapting to the visual theme and aesthetic of your brand. This includes turnkey support for light and dark mode right out of the box.

![image](https://github.com/skeletonlabs/skeleton/assets/1509726/084d8912-30c6-404a-adeb-555c3e040709)

Plus each template is bundled with it's own unique hand-crafted theme, which you may reuse in any number of projects! You can also opt for any of Skeleton's preset themes or implement your own using our powerful Theme Generator. Add your own images and content and you're ready to deploy!

## Functional Apps

Our templates also operate as fully functional web applications. Take the blog template for example – this implements mdsvex, which provides a simple to use yet powerful Markdown blogging system. If you wish to create a new post, just add a new file, no CMS or backend required. This template also comes with type-safe frontmatter, a customizable category system, and support for multiple authors. 

![image](https://github.com/skeletonlabs/skeleton/assets/1509726/87a88cb5-a504-4d7a-a153-ff652e8b150c)

The Portfolio template provides a local configuration file to quickly and easily customize the details of each listed project and add or edit the category grouping. With all routing handled for you automatically.

![image](https://github.com/skeletonlabs/skeleton/assets/1509726/3f8abb13-fa5d-4e3c-868b-cf5eb519743a)

## Best in Class Features

Each template is generated using Skeleton, which means it's powered by Svelte, SvelteKit, and Tailwind under the hood. Use Skeleton's vast array of components and elements, Tailwind's powerful utility classes, or swap your SvelteKit adapter for pain-free deployment.

### Typescript Support

Full support for Typescript's type safety by default. Ensuring you can easily consume and extend data with confidence.

![image](https://github.com/skeletonlabs/skeleton/assets/1509726/4e2b42b5-de49-485c-9bbc-aa8a7a301f16)

### Custom Components

We've included a number of custom features as well, such as a Social Links component. This allows you to easily configure the links and icons for social networks. Font Awesome is included by default to handle icons.

![image](https://github.com/skeletonlabs/skeleton/assets/1509726/84e08dce-bd4a-4f0d-8e2a-f8455d119dfc)

### Sandbox

Each template features a handy sandbox page for previewing your theme styling, testing Skeleton features, or implementing your own custom components. This can be found from the hidden /sandbox route.

![image](https://github.com/skeletonlabs/skeleton/assets/1509726/ca7fcd5e-fae4-4ffa-a895-d7ebe5f872b1)

### Free Updates

All templates are maintained to support the latest and greatest versions of Svelte/Kit, Tailwind, and Skeleton. Including support for Svelte v4 out of the box, as well as Skeleton v2 in the near future.

## Licensing

When you purchase a template from the [Skeleton Store](https://store.skeleton.dev/) you have access to several flexible license options. We've taken great care to tailor these for a developers at any scale, including:

* Personal - intended for personal project, hobbyists, and freelancers.
* Commercial - for small to medium size businesses, including agencies.
* Enterprise - for large organizations that need custom rights for distribution.

The Personal and Commercial license options are available when adding each product to the cart on the [Skeleton](https://www.skeleton.dev/) Store, while users interested in our Enterprise license may contact Skeleton Labs at admin@skeleton.dev.

## Curated by Skeleton Labs

Each template has been designed and implemented by myself (Chris Simmons) – the creator and core maintainer of Skeleton, as well as co-founder of Skeleton Labs. I've had the privilege of serving as a professional frontend developer for a little over two decades. I go to great lengths to ensure each project meets the highest standards of quality and follow the best practices for Skeleton.

![image](https://github.com/skeletonlabs/skeleton/assets/1509726/98240de4-6a61-45e4-8f1a-27f6a583f2a8)

> As an aside - we will be looking for new creators in the future! If you are a professional frontend developer with a keen eye for design and strong emphasis on high quality UX, please contact me at chris@skeleton.dev and let's talk!

## Who Benefits?

As I'm sure you're aware, Skeleton is a free and open source library. The Skeleton project is tended by Skeleton Labs, which offers a number of [paid services](https://www.skeletonlabs.co/) and sponsorship through [Ko-Fi](https://ko-fi.com/skeletonlabs), [GitHub](https://github.com/sponsors/skeletonlabs), and [Patreon](https://www.patreon.com/user?u=83786276). All funds from these services are aimed at covering Skeleton's hosting fees, third party services, contractors, and other expenses.

However, the Skeleton Store is a bit different. Each purchase will go directly to the author of the product. So for example, any templates you purchased today will support me directly! As you may know, I've been a full time contributor to Skeleton since it's inception over a year and half ago. I have served as the core maintainer since the project went open source in August 2022 and Skeleton is my full time job. My goal is to continue to build and maintain Skeleton for as long as I'm able, so each purchase you make helps enable this.

Funding development of a large open source project is a real challenge, but I feel this is the most ethical optional available to do so - plus you get really cool products in return, which I hope is a win for everyone!

## What's Next?
Currently we're aiming to introduce a new template each month. This may include e-commerce and dashboard templates in the future. We're also looking to introduce entirely new product categories as relevant. Of course we're also excited to bring the current slate of goods to other countries as soon as possible!

That said, please stay tuned for further announcements, and as always, thanks to everyone for your support! We wouldn't be here with you.

---

> **Note** - Enter code launch at checkout to received 10% off your first purchase! Available until August 1st, 2023.

View the newly launched Skeleton Store at [store.skeleton.dev](https://store.skeleton.dev)

**Comment by delight** [comment]
Great news ! 

One little note: The link "_View the newly launched Skeleton Store at [store.skeleton.dev](https://github.com/skeletonlabs/skeleton/discussions/store.skeleton.dev)_"  leads to https://github.com/skeletonlabs/skeleton/discussions/store.skeleton.dev instead of [store.skeleton.dev](https://store.skeleton.dev/)

**Reply by endigo9740** [reply]
Fixed! :)

**Comment by mmailaender** [comment]
Have you considered an all-access licence like Tailwind does? Highly interested into it https://tailwindui.com/all-access

**Reply by endigo9740** [reply]
Oh definitely, we've been crunching all kinds of numbers behind the scenes! What you see today is essentially the bedrock foundation and we expect a lot to grow and change over the coming months. I'm not sure if the all-access approach will be right for us (we're no where near as big as Tailwind Labs), but I am interested in more affordable access to features in bulk. We want folks to have easy access to our content for sure. What that looks like is still to be determined, but might take the shape of a subscription, bundles, etc.

**Comment by mmailaender** [comment]
Is there already some rough timeline when the store will be available outside of US? 🙂
![Screenshot_20230818-072148.png](https://github.com/skeletonlabs/skeleton/assets/87228994/a97decc0-e9d1-4374-ab49-2571f1ca14ce)

**Reply by endigo9740** [reply]
Roll out will depend on the country or region.  We're anticipating Canada and various European countries to be up first though. v2 is dominating my full attention right now, so it'll definitely be after that.

**Reply by mmailaender** [reply]
Do you have some particular challenges here? It's the first time that I see such a notification for a digital product.

**Reply by endigo9740** [reply]
I don't think the majority of small companies adhere to, or maybe even understand tax law differs from country to country. A lot likely forego it and hope for the best. In order to sell digital goods in countries like the US, you have to have register for a tax ID within your nexus state. It took sitting down and filing an online form to apply for an receive this.

For other countries, it comes down to learning what the local requirements are for tax law, applying for whatever is needed, getting your tax ID, then registering that information in our online shop (Shopify). It's a very manual process and this is being done by essentially one person for Skeleton Labs (myself).

Some groups may think it's fine to forego tax requirements for the countries they are selling in. I do not. Simple as that.

**Reply by mmailaender** [reply]
Do you know the [MoR (Merchant of Record)](https://www.paddle.com/resources/merchant-of-record-a-guide-for-cfos) concept? This is one solution that I can think about, that could solve your tax challenge.

If you want to continue your shop as is, probably [Paddle](https://www.paddle.com/) could be a good solution.
If you have thought about providing the website templates as a subscription you could solve it also with GitHub monthly sponsoring.

Haven't used the tools on my own, but both of the products are used, e.g., from [OSS TipTap](https://tiptap.dev/).

----
Edit:
It seems Shopify is currently also creating something like that. Have you already seen that? Unfortunately, it's in early access - only for selected merchants. [Eventually, your account is eligible? ](https://admin.shopify.com/settings/markets/pro/pre-application)

**Reply by endigo9740** [reply]
I was not actually. I'm definitely firmly in the arena of developer and not tax professional. I was interesting in bringing in a tax professional to help with some of this at some point though, but right now I don't believe that's a viable option. I appreciate you bring this to my attention though and it's interesting to hear Shopify has a solution incoming for this.

Do you have the name for the Shopify service by chance? The link you provided was not loading for me.

> If you have thought about providing the website templates as a subscription you could solve it also with GitHub monthly sponsoring.

Interesting suggestion, I'll consider this. I have always been keen to provide more in return for sponsorship and donations. This might be a great way to go about that. The problem is the logistics of delivery. I'll look into the mechanisms provided by each of our services though to see if this is possible.

**Reply by mmailaender** [reply]
Shopify calls it [Markets Pro](https://www.shopify.com/markets). You need to scroll down to the section "Add Markets Pro to expand faster".

![image](https://github.com/skeletonlabs/skeleton/assets/87228994/b692e857-0d5d-4644-82f1-e44d447767da)

**Reply by endigo9740** [reply]
That's awesome, I'll keep an eye on this. In the interum we still likely be sticking with Shopify and facing the issues mentioned above. But again, I'm hoping after v2 ships I can devote more attention to this again.
